src/Image.tsx

```diff
@@ -4,18 +4,18 @@
  * @author github.com/gokcan
  */
 
-import React, { ReactNode } from 'react'
+import React, { ReactNode, ImgHTMLAttributes, Component } from 'react'
 import PropTypes from 'prop-types'
 
 import IntendedError from './IntendedError'
 
-interface Props {
+export interface ImageProps {
```

Comment:
Do we export `ImageProps` at where we import `Image.tsx`? It would be a good idea if we export the component props to the user of this package.

Reply:
Frankly I've no experience with rollup, but with this change it should be possible to do exactly that: `import Image, { ImageProps } from 'react-shimmer';`.

Reply:
Thank you!